### ISSUE TITLE
[EN][Engineering] Flexweave underlay

### ISSUE THREAD

**Issue by zotmer** [Atlantiss/BugTracker#7055]
Flexweave underlay (http://www.wowhead.com/spell=55002/flexweave-underlay#comments) summons a parachute to slow falling speed of an engineer. However, the parachute should cancel the moment you hit the ground. 

On Atlantiss, you have it the whole duration (30 seconds) even if you hit the ground.
You can even mount with it, creating some hilarious mount parachuted moments.

In addition to that, you should be unable to use Flexweave underlay while on the ground.

Video proof of how it should work: https://www.youtube.com/watch?v=uHsfcRPCoSs (0:14)

Proof from Atlantiss: ![flexweave_underlay](https://cloud.githubusercontent.com/assets/18169265/14211770/2238d844-f830-11e5-9e08-50cd162cbd96.jpg)

**Comment by fruvel** [comment]
yep. u can use this on ground, mount, and fly while battleground (eye of the storm) starts :) it's epic bug :P

**Comment by ShockRW** [comment]
BUMP
